raiden/ui/console.py

```diff
@@ -45,19 +56,19 @@ def print_usage():
     print("\n" + ENDC)
 
 
-def inputhook_gevent():
+def inputhook_gevent() -> int:
     while not stdin_ready():
         gevent.sleep(0.05)
     return 0
 
 
 @inputhook_manager.register("gevent")
 class GeventInputHook:
```

Comment:
This whole class seems to be unused. Can you remove it please?

Reply:
I thought the same, but as it gets registered I wasn't sure that this isn't used somehow.